src/components/PDOKAutoSuggest/index.js

```diff
@@ -55,10 +55,13 @@ PDOKAutoSuggest.propTypes = {
   className: PropTypes.string,
   fieldList: PropTypes.arrayOf(PropTypes.string),
   /**
-   * Value that determines to which municipality the search query should be applied
-   * Can be a single name, like amsterdam, or a combination, like (amsterdam OR weesp)
+   * Value that determines to which municipality the search query should be
+   * applied.
+   *
+   * Can be a single name, like amsterdam, or an array of one or more names,
+   * which will be combined using a logical OR
```

Comment:
There isn't a construct in the code that point to the use of the logical OR. Is it worth mentioning this?

Reply:
To make clear that that's the way multiple municipalities will be logically combined, with OR, not AND. But maybe you're right, wouldn't make sense to combine them with AND, so unnecessary to mention it.